languages/scala/exercises/concept/operators/.docs/instructions.md

```diff
@@ -0,0 +1,60 @@
+In this exercise you're going to write a calculator for rectangles.
+
+Given is a case class `Rectangle`. A rectangle is defined by its width and its height:
+
+```scala
+case class Rectangle(width: Int, height: Int)
+```
+
+The calculator will allow to sum up two rectangles and to increase a rectangle by multiplying it with a number.
+
+## 1. Create a plus operator for rectangles
+
+Implement a `+` method that takes another rectangle as argument. Assume that the passed rectangle has the same height as the current one. The method should return a new rectangle, whose width is a sum of both rectangle's widths and whose height is the same as of both rectangles.
+
+Let's visualize the expected outcome:
+
+![sum of 2 rectangles of same height](sum_same_height.png)
```

Comment:
This exercise uses an image to make things clearer, which we usually don't do in exercise. There are several reasons for this, including images not always being suitable for different form factors. @iHiD have we made a decision on using images in exercises?

I think that in this case, the description together with the example might be good enough to allow not using an image.

Reply:
Yeah - we can't have images as these need to work when you download it and read it just as markdown. Also it would break accessibility if the image was critical to the exercise.

Reply:
Ah, what a pity. Ok fine, I will throw them out.